contrib/DEBIAN/copyright

```diff
@@ -77,6 +78,331 @@ License: PUB-DOM
 Copyright: Crobbo (forum)
 Comment: Site: https://bitcointalk.org/index.php?topic=32273.0
 
+Files: depends/sources/libsodium-*.tar.gz
+Copyright: 2013-2016 Frank Denis
+License: ISC
+
+Files: depends/sources/libsnark-*.tar.gz
+Copyright: 2012-2016 SCIPR Lab and contributors; 2016 Zcash developers
```

Comment:
No, I distinctly remember us deciding not to use "Zcash Core" in licensing. So "2016 The Zcash developers"